diff --git a/engine/engine.py b/engine/engine.py
index f78212b..a6d5e01 100644
--- a/engine/engine.py
+++ b/engine/engine.py
@@ -224,6 +224,7 @@ class Engine(ibus.EngineBase):
         self.__set_dict_mode_props(anthy_props)
         self.__set_dict_config_props(anthy_props)
         anthy_props.append(ibus.Property(key=u"setup",
+                                         icon=u"preferences-desktop",
                                          tooltip=UN(_("Configure Anthy"))))
 
         return anthy_props
From 70633cda85851e0cdd6353d553649cbb51200a5c Mon Sep 17 00:00:00 2001
From: fujiwarat <takao.fujiwara1@gmail.com>
Date: Fri, 10 Dec 2010 14:54:15 +0900
Subject: [PATCH] Fixed the latest page_size for ibus.LookupTable.

---
 engine/engine.py |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/engine/engine.py b/engine/engine.py
index c2bc8f0..6dd4461 100644
--- a/engine/engine.py
+++ b/engine/engine.py
@@ -727,6 +727,9 @@ class Engine(ibus.EngineBase):
         self.__refresh_typing_mode_property()
 #        self.__reset()
 #        self.__invalidate()
+        size = self.__prefs.get_value('common', 'page_size')
+        if size != self.__lookup_table.get_page_size():
+            self.__lookup_table.set_page_size(size)
 
     def focus_out(self):
         mode = self.__prefs.get_value('common', 'behavior_on_focus_out')
-- 
1.7.5.4

